doc/devguide.md

```diff
@@ -16,32 +18,20 @@ Make sure the following dependencies are installed in your environment before yo
 
 Besides using Visual Studio in Windows, you can build by running the build script:
 
-### Linux
 ```sh
 scripts/linux/buildBranch.sh
 ```
 
-### Windows
-```powershell
-scripts\windows\buildBranch.bat
```

Comment:
Do we want to keep this for local dev on windows? How would the dev story look like on windows when we remove all windows support? does it mean that going forward we have to use WSL(2) for local dev?

Reply:
I talked to Denis about this. People can still dev on windows as they do now. They just cannot run the checkins locally (did anyone even do this?).

I brought up keeping this and other windows files, but feel that it would be inconsistent with the new checkins we are adding (i.e. watchdog) which only have linux supported builds.